src/test/java/spoon/support/reflect/CtModifierHandlerTest.java

```diff
@@ -0,0 +1,67 @@
+package spoon.support.reflect;
+
+import org.junit.Before;
+import org.junit.Test;
+import spoon.Launcher;
+import spoon.reflect.declaration.ModifierKind;
+import spoon.reflect.factory.Factory;
+
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.*;
+
+public class CtModifierHandlerTest {
+
+    LinkedHashSet<ModifierKind> inputModifierKinds = new LinkedHashSet<>(Arrays.asList(
+            ModifierKind.STATIC,
+            ModifierKind.PUBLIC,
+            ModifierKind.ABSTRACT,
+            ModifierKind.NATIVE
+    ));
+    LinkedHashSet<CtExtendedModifier> inputExtendedModifiers = inputModifierKinds.stream().map(CtExtendedModifier::new).collect(Collectors.toCollection(LinkedHashSet::new));
+
+    // these will be overwritten in each test.
+    Set<ModifierKind> modifiers;
+    Set<CtExtendedModifier> modifiersExt;
+    CtModifierHandler handler;
+
+    @Before
+    public void setUp() {
+        Launcher l = new Launcher();
```

Comment:
Why not name them `launcher` and `factory`? One letter names make this code here a lot harder to read than would be necessary.